tabbycat/draw/templates/draw_display_admin.html

```diff
@@ -211,8 +211,11 @@ <h5 class="card-title mb-0">{{ release_motions_title }}</h5>
 
           <form id="releaseMotionsForm" method="POST" hidden
                 action="{% roundurl 'motions-release' %}">{% csrf_token %}</form>
-
-          {% if pref.participant_ballots != 'off' and pref.enable_motions %}
+          {% if not round.motion_set.exists %}
+            {% trans "No motion(s) are currently set for this round." as text %}
+            {% include "components/item-info.html" with nopad=True type="warning" %}
+            {% include "components/item-action.html" with type="success" id="triggerReleaseMotionsForm" text=release_text url="" to_complete=True %}
```

Comment:
Either we should try to avoid duplicating this button (as it is the same in the `elif`), or it should be in a warning state and indicate the problem (and why it is) in the text.